app/grants/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.contrib.humanize.templatetags.humanize import naturalday, naturaltime
+from economy.models import SuperModel
+
+
+class Grant(SuperModel):
+    """Define the structure of a Grant."""
+
+    status = models.BooleanField(default=True)
+    title = models.CharField(default='', max_length=255)
+    description = models.TextField(default='', blank=True)
+    reference_url = models.URLField(db_index=True)
+    image_url = models.URLField(default='')
```

Comment:
Where is this image stored? If it's in our app, we'll want to update this to be an `ImageField` (`FileField` if we anticipate this will be an SVG).